ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -280,7 +280,7 @@ - (RTCVideoTrack *)createScreenCaptureVideoTrack {
         [devices addObject:@{
             @"facing" : position,
             @"deviceId" : device.uniqueID,
-            @"groupId" : @"",
+            @"groupId" : device.deviceType,
```

Comment:
I don't think the device type matches well the group concept, please leave it out.